src/averageAges.js

```diff
@@ -55,7 +67,19 @@ function calculateWomenAverageAge(people, withChildren) {
  * @return {number}
  */
 function calculateAverageAgeDiff(people, onlyWithSon) {
-  // write code here
+  let children = people.filter(child =>
+    child.mother && people.find(person =>
+      person.name === child.mother));
+
+  children = onlyWithSon
```

Comment:
for this (70 & 74 lines), it is better to use different variables, for example `filteredChildren` and `children`